Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM node:alpine as build
+FROM node:12-alpine as build
 WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm install
 COPY . .
-RUN npm install && npm run prodbuild
+RUN npm install -g @angular/cli
+RUN ng build --configuration production --output-path=/dist
```

Comment:
Was wondering if this change was necessary?  Do we need to change the output path here?  Otherwise we could just leave it as the default 'npm run prodbuild'

Reply:
Oh no you're right we definitely don't, I can change that